service/internal/pipelines/pipelines.go

```diff
@@ -304,6 +304,21 @@ func Build(ctx context.Context, settings component.TelemetrySettings, buildInfo
 	return exps, nil
 }
 
+func logStabilityMessage(logger *zap.Logger, sl component.StabilityLevel) {
```

Comment:
Should we have a "InfoMessage" (or find a better name) on `component.StabilityLevel` (can even accept zap.Log or return the string) to help with this? 

Reason: you forget to do the same in `extensions` package, where you will need to duplicate this.

Reply:
Sounds good, I thought about it and ended up choosing to go this route, but you're right we'll need the same thing for extensions.

I'll follow up this PR with a fix for this that can be used in extensions